This patch just silences a warning about implicit declaration of setupColors
--- ../objects.h.orig	2020-05-09 17:21:04.179875753 +0200
+++ ./objects.h	2020-05-09 17:20:45.526561303 +0200
@@ -49,4 +49,6 @@
 /* Checks for collision between a player object and a target */
 int collision (struct Object player, struct Object target);
 
+void setupColors ();
+
 #endif
